src/api/worker/rest/DefaultEntityRestCache.ts

```diff
@@ -704,6 +710,9 @@ export class DefaultEntityRestCache implements EntityRestCache {
 		// No need to try to download something that's not there anymore
 		if (cached != null) {
 			try {
+				// if the password changed we'll be logged out at this point.
+				// not catching the NotAuthorized or NotAuthenticated Error that results of this to delete the user
```

Comment:
>not catching the NotAuthorized or NotAuthenticated Error that results of this to delete the user

I am confused by the meaning and the sentence itself

Reply:
wording needs to be improved

we forgot to mention that this is true for user updates. it was supposed to mean that we will drop into the session recovery flow if we don't catch that (expected) error here